test/runtests.jl

```diff
@@ -91,6 +91,16 @@ prepend!(tests, linalg_tests)
 
 import LinearAlgebra
 cd(@__DIR__) do
+    # `net_on` implies that we have access to the loopback interface which is
+    # necessary for multithreading. There are some build environments that do
```

Comment:
loopback access and multithreading are unrelated. Without network access what is disallowed is the TCP based multiprocessing that is used for the test suite (though somebody should probably just add an option to those over domain sockets instead)

Reply:
I think this should be resolved by 4cf735b now. Feel free to let me know if not!